Core/Entities/Role.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace Core.Entities
+{
+    public class Role : IdentityRole<Guid>
```

Comment:
Mark this class with a comment explaining that it's necessary to make IdentityCore function. It looks strange by itself